src/main/java/vention/pages/HomePage.java

```diff
@@ -0,0 +1,88 @@
+package vention.pages;
+
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.By;
+
+public class HomePage extends BasePage {
+  private static final String HOME_PATH = "/travel/home/public/en_us/";
```

Comment:
I'd recommend to rename relative path variable to have the same and generic name in every page class, ex. RELATIVE_PATH

It would help with further maintainability